terraform/stacks/main/outputs.tf

```diff
@@ -103,30 +103,30 @@ output "services_subnet_az2" {
   value = "${module.cf.services_subnet_az2}"
 }
 output "services_subnet_cidr_az1" {
-  value = "${var.services_cidr_1}"
+  value = "${module.cf.services_cidr_1}"
 }
 output "services_subnet_cidr_az2" {
-  value = "${var.services_cidr_2}"
+  value = "${module.cf.services_cidr_2}"
 }
 output "services_subnet_gateway_az1" {
-  value = "${cidrhost("${var.services_cidr_1}", 1)}"
+  value = "${cidrhost("${module.cf.services_cidr_1}", 1)}"
```

Comment:
I see that this issue predates your patch, but I don't think we need all the quotes and curly braces here--I think we can use

```
value = "${cidrhost(module.cf.services_cidr_1, 1)}"
```